modules/gapi/include/opencv2/gapi/infer/ie.hpp

```diff
@@ -150,6 +162,16 @@ template<typename Net> class Params {
         return *this;
     }
 
+    Params& contextConfig(cv::util::any&& ctx_cfg) {
```

Comment:
We have a different naming model here (see other methods), so maybe it'd better look like `cfgContext`.

Reply:
There is `pluginConfig` just atop of it which in my opinion does rather the same. Do I need to rename it too?

Reply:
Ok, will rename this new config as `cfg...`

Reply:
Renamed